package.json

```diff
@@ -37,7 +37,7 @@
     "winston": "^3.1.0"
   },
   "devDependencies": {
-    "async": "^2.6.1",
+    "async": "^3.2.3",
```

Comment:
You also need to update this in the regular dependencies section since `ocsp` relies on `async` and the `ocsp` package is pulling a bad version of `async` which is causing the build failure.